FRCs/frc-00zz.md

```diff
@@ -0,0 +1,73 @@
+## Simple Summary
+Timelock encryption is a method of encrypting a message now that can't be decrypted until a specific point in the future has passed. The drand team have launched a scheme that uses BLS12-381 signatures in an innovative way using [identity based encryption](some-link-here) to achieve practical Timelock encryption using the drand network.
```

Comment:
```suggestion
Timelock encryption is a method of encrypting a message now that can't be decrypted until a specific point in the future has passed. The drand team has launched a scheme that uses BLS12-381 signatures in an innovative way using [identity based encryption](some-link-here) to achieve practical Timelock encryption using the drand network.
```

Reply:
Both `have` and `has` are acceptable for collective nouns